tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -138,6 +238,36 @@ iface_namingmode/test_iface_namingmode.py::TestShowQueue::test_show_queue_waterm
     conditions:
       - "platform in ['x86_64-cel_e1031-r0']"
 
+#######################################
+#####            ip               #####
+#######################################
+ip/test_ip_packet.py:
+  skip:
+    reason: "Skipping ip packet test since can't provide enough interfaces"
+    conditions:
+      - "len(minigraph_interfaces) < 2 and len(minigraph_portchannels) < 2"
```

Comment:
Are you sure these variables like `minigraph_interfaces` and `minigraph_portchannels` are available for evaluation?